src/models/sunburst.js

```diff
@@ -59,6 +59,15 @@ nv.models.sunburst = function() {
 
             var g = wrapEnter.selectAll('nv-sunburst');
 
+            chart.update = function() { 
+              if( duration === 0 )
+                 container.call(chart);
+               else
+                 container.transition().duration(duration).call(chart);
```

Comment:
Please make sure the indention is 4 spaces... the "if" and "else" aren't even lined up here...   

Also, please use brackets for if/else in call cases to match the rest of the code style that we're going for (jslint/jshint rules).